fec/home/views.py

```
@@ -37,6 +38,8 @@ def get_press_releases(category_list=False, year=False):
   return press_releases
 
 def updates(request):
+  # Only render view if the user is authenticated or there's a feature flag
+  if request.user.is_authenticated() or settings.FEATURES['latest_updates']:
```

Not a huge deal, but I prefer the "fail fast" style functions because they're easier to read. Avoids having a chunk of indented code in if/else blocks.
```python
def updates(request):
  if not (request.user.is_authenticated() or settings.FEATURES['latest_updates']):
    return render(request, '404.html')

  # render code here
```